### ISSUE TITLE
Content: ALT text

### ISSUE THREAD

**Issue by TedGoas** [StackExchange/Stacks#395]
Sometimes our designs include images. All images should have an `alt` attribute, but what goes in there is important for folks using assistive technologies or plain text preferences.

It would be helpful to have a framework for us to write consistent, helpful `alt` text. There are a few unofficial, undocumented best practices floating around, let's organize and document them.
